docs/plot.rst

```diff
@@ -0,0 +1,372 @@
+
+- compare generation and capacity in various scenarios through pie charts (`notebook
+  <https://github.com/Breakthrough-Energy/PostREISE/blob/develop/postreise/plot/demo/pie_generation_vs_capacity_demo.ipynb>`_)
+
+  .. code-block:: python
+
+      from postreise.plot.plot_pie_generation_vs_capacity import (
+        plot_pie_generation_vs_capacity,
+      )
+      from powersimdata.utility.helpers import PrintManager
+
+      with PrintManager():
+          plot_pie_generation_vs_capacity(
+              areas=["WA", "Western"],
+              scenario_ids=[2497, 3101],
+              scenario_names=[
+                  "Western 90% clean and 10% nuclear",
+                  "Western 90% clean and 10% nuclear \n with transmission upgrade",
+              ],
+          )
+  .. image:: img/comp/capacity_vs_generation_wa_pie.png
+  .. image:: img/comp/capacity_vs_generation_western_pie.png
+
+- compare generation shortfall in various scenarios through bar charts (`notebook
+  <https://github.com/Breakthrough-Energy/PostREISE/blob/develop/postreise/plot/demo/bar_shortfall_demo.ipynb>`_)
+
+  .. code-block:: python
+
+      from postreise.plot.plot_bar_shortfall import plot_bar_shortfall
+      from powersimdata.design.generation.clean_capacity_scaling import (
+        load_targets_from_csv,
+      )
+      from powersimdata.utility.helpers import PrintManager
+
+      with PrintManager():
+          target_df = load_targets_from_csv(
+              "postreise/data/2030_USA_Clean_Energy_Regular_Targets.csv"
+          )
+          plot_bar_shortfall(
+              "Nevada",
+              [2497, 3101],
+              target_df,
+              scenario_names=[
+                  "Western 90% clean and 10% nuclear",
+                  "Western 90% clean and 10% nuclear"
+                  + "\n"
+                  + "with transmission upgrade",
+              ],
+              baseline_scenario=2497,
+              baseline_scenario_name="Western 90% clean and 10% nuclear",
+          )
+
+  .. image:: img/comp/shortfall_nv.png
+
+
+Emission
+########
+- compare total carbon emissions by generator type for 1-to-n scenarios through bar
+  charts (`notebook <https://github.com/Breakthrough-Energy/PostREISE/blob/develop/postreise/plot/demo/emissions_map_demo.ipynb>`_)
+
+  .. code-block:: python
+
+      from postreise.plot.plot_carbon_bar import plot_carbon_bar
+      from powersimdata import Scenario
+      from powersimdata.utility.helpers import PrintManager
+
+      scenarioA = Scenario(2497)
+      scenarioB = Scenario(3101)
+
+      with PrintManager():
+          scenarioA = Scenario(2497)
+          scenarioB = Scenario(3101)
+
+          plot_carbon_bar(
+              scenarioA, scenarioB,
+              labels=["Western" + "\n" + "90% clean and 10% nuclear",
+                      "Western" + "\n" + "90% clean and 10% nuclear" + "\n" + "with
+                       transmission upgrade"]
```

Comment:
For a better copy-paste experience, move
`+ "with transmission upgrade"]` to its own line
similar to the chart names one chart above this one